.github/workflows/helm_release.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
`fetch-depth` can increase the time CI takes to clone the repo.

Does chart-releaser need the commit info to produce a release?

If not, I prefer if this input is removed.

Reply:
@vyas-n I'm not completely sure on this. All the examples of using chart-releaser have had fetch-depth set to `0`. I can do some testing on my fork to confirm.

Reply:
@vyas-n Tested with out `fetch-depth: 0` here and received an error:

https://github.com/joshsizer/docker-registry.helm/actions/runs/8429788969/job/23084576246#step:5:12

```shell
fatal: invalid reference: origin/gh-pages
Error: exit status [12](https://github.com/joshsizer/docker-registry.helm/actions/runs/8429788969/job/23084576246#step:5:13)8
```

Reply:
Oh my bad, I didn't realize it used the checkout git repo directly to commit & push the changes to the `gh-pages` branch.